src/main/java/com/fuzs/aquaacrobatics/integration/IntegrationManager.java

```diff
@@ -35,6 +37,7 @@ public static void loadCompat() {
         isHatsLoaded = Loader.isModLoaded("hats");
         isThaumicAugmentationLoaded = Loader.isModLoaded("thaumicaugmentation");
         isTrinketsAndBaublesLoaded = Loader.isModLoaded("xat");
+        isWitcheryResurrectedLoaded = Loader.isModLoaded(WitcheryResurrected.MOD_ID);
```

Comment:
I think you have to use `"witchery"` here; the `WitcheryResurrected` class won't exist at runtime if the mod isn't present, and that will cause a ClassNotFoundException.